MAUI/Cartesian-Charts/Range-Column.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Range Column Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about range column and range bar chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Range Column Chart in .NET MAUI Chart
+
+## Range Column Chart
+
+* Range Column Chart is used to visualize data points with columns, with the height of each column representing the difference between the lowest and highest values of the data point. 
+
+* To render a range column chart, create an instance of [RangeColumnSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```

Comment:
* To render a range column chart, create an instance of the [RangeColumnSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).